src/components/forms/AppDateInput.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
I'm not sure about this component. I tried something like this for the callout editor and found that it was much more reliable just to keep the date in text YYYY-MM-DD then convert before sending to the API.

Arguably there might be a place for an `AppDateTimeInput` which could handle converting the `Date` to a date/time string and vice versa but for now I think you should follow the pattern used in the callout edit code